contracts/acl/ACLSyntaxSugar.sol

```diff
@@ -86,6 +86,31 @@ contract ACLSyntaxSugar {
 
 
 contract ACLHelpers {
+
+    enum Op { NONE, EQ, NEQ, GT, LT, GTE, LTE, RET, NOT, AND, OR, XOR, IF_ELSE } // op types
```

Comment:
What do you think about putting this in another contract, e.g. `ACLParams`?

Reply:
Good idea! I agree they fit more inside `ACLParams`